include/mxnet/lib_api.h

```diff
@@ -229,20 +241,41 @@ enum MXReturnValue {
   MX_SUCCESS = 1,
 };
 
+struct MXSparse {
+  // Pointer to data.
+  void *data{nullptr};
+  // length of (non-zero) data.
+  int64_t data_len;
+
+  // To store aux data for sparse.
+  // For CSR, indices stores the col index of non-zero values.
+  // For row sparse, indices store row index of rows which have non-zero values.
+  std::vector<int64_t> indices;
+
+  // For CSR, indptr gives the start and end index of data for each row.
+  // For row sparse, indptr is empty. 
+  std::vector<int64_t> indptr;
+
+  // Context of data.
+  // MXContext ctx;
```

Comment:
we can remove this since theres a context in the MXTensor right?

Reply:
Removed. Thanks!